fish-rust/src/ffi.rs

```diff
@@ -115,6 +138,12 @@ impl From<wcharz_t> for wchar::WString {
     }
 }
 
+impl Debug for re::regex_t {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        f.write_str("regex_t")
+    }
+}
```

Comment:
Should I keep this? Need it for impl Debug on Abbr

Reply:
seems fine to keep